### DISCUSSION TITLE
Customer Specific Pricing

### DISCUSSION THREAD

**Discussion by erikfritzler** [saleor/saleor#11908] [General]
I am trying to figure out if it would be feasable to setup customer specific pricing in Saleor.
If I have a customer who gets a different price on a product. They would see that price in the online store.

**Comment by CossackDex** [comment]
Hi @erikfritzler

Customer-specific pricing is feasible by using Customer Metadata and Checkout custom prices. You can read more about Checkout custom prices here https://docs.saleor.io/docs/3.x/developer/checkout#setting-custom-prices-for-checkout-items and about Metadata here https://docs.saleor.io/docs/3.x/developer/metadata .

**Comment by timuric** [comment]
@erikfritzler we are planning to release customer groups pricing around April
